content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2246,19 +2254,19 @@ Manually defining an `@@index` in the Prisma schema serves no purpose.
 | Name     | Required | Type               | Description                                                                                                                                                                                                                                 |
 | -------- | -------- | ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `fields` | **Yes**  | `FieldReference[]` | A list of field names - for example, `["firstname", "lastname"]`                                                                                                                                                                            |
-| `name`   | **No**   | `name`             | The name of the index in the underlying database (Prisma generates an index name that respects identifier length limits if you do not specify a name. Prisma uses the following naming convention: `tablename.field1_field2_field3_unique`) |
+| `map` ([2.27.0](https://github.com/prisma/prisma/releases/tag/2.27.0) and later)  | No   | `String`             | The index name in the underlying database. Default naming convention: `{table_name}_{column_names}_idx` (for example, `users_col1_col2_idx`) for globally unique indexes and `{column_names}_idx` for indexes that are only unique at table level (for example: `col1_idx1`) |
 
 The _name_ of the `fields` argument on the `@@index` attribute can be omitted:
 
 ```prisma no-lines
 @@index(fields: [title, author])
-@@index([title, author])
+@@index([title, author], map: "MyConstraintName")
```

Comment:
This makes it look a bit like the omitting was possible because the `map` was added - maybe a separate example for `map`?